k8s.gcr.io/images/k8s-staging-artifact-promoter/images.yaml

```diff
@@ -12,19 +12,19 @@
     "sha256:768443d88a1c2732adbccdf2d633c35f5a658d92ebf500d0637e2b686e6c239a": ["20200825-v2.3.1-262-g1d9ded6"]
     "sha256:799f81d9a855717eab7198310655d1b55919320b120f0f015c3bb43c4e90316d": ["20190821-v2.2.1-2-gc29e4fc"]
     "sha256:7c672405701dc7decdf98730708506cc64f502a965893cdb441e4943439b6d28": ["20200107-v2.3.1-71-gfd2fae8"]
-    "sha256:883e37aa7651b801a2ed55afc774c2b06caa8756a84f43ed5632dc36a92984c0": ["20201027-pkgv0.3.0-7-gf131ef2", "v2.4.1"]
+    "sha256:883e37aa7651b801a2ed55afc774c2b06caa8756a84f43ed5632dc36a92984c0": ["20201027-pkgv0.3.0-7-gf131ef2","v2.4.1"]
```

Comment:
Interesting...

Reply:
was this file authored manually before?

Reply:
> was this file authored manually before?

Some aspects of it initially, but this is one of the few manifests that has been using `cip-mm` for a while now.
I _think_ what's happening is the grow manifest functionality is stripping spaces in instances where there are multiple tags.

Reply:
xref: https://github.com/kubernetes/k8s.io/pull/3212#discussion_r771604521